writings/2021-12-12-the-importance-of-the-most-profitable-block.mdx

```diff
@@ -0,0 +1,86 @@
+
+Moreover, Flashbots Alpha auctions off block space by optimizing for bundles with the highest gas price. Unfortunately this is not the same as optimizing for overall bundle profit. Consider the following two bundles:
+* Bundle A: 100,000 gas with miner payment of 1 ETH and gas price of 10,000 GWEI
+* Bundle B: 1,000,000 gas with miner payment of 9.9 ETH and gas price of 9,900 GWEI
+
+Bundle A will win the Flashbots Auction because it has a higher gas price despite bundle B being more profitable overall for miners.
+
+The reason this design decision was made is because optimizing bundles for gas price is computationally simpler than optimizing for overall profit. However this opens up a dimension for competition: if a miner has a proprietary solution that allows them to optimize bundles for overall profit they could extract more MEV than their peers in some cases. Similarly, being able to include more bundles per block would also lead to more MEV extraction.
+
+Lastly, in Flashbots Alpha bundles are seen by both relays and miners. As a result searchers need to make a key trust assumption that relays and miners won’t frontrun or unbundle their bundles. This may be an acceptable assumption for searchers to make with large and established miners who have social capital, but not with small and newer miners who lack social capital. It also opens up an attack vector whereby a searcher could run a miner in order to receive bundle flow themselves, which is why Flashbots needs to require that miners have a history of mining prior to sending them bundles. Moving from proof-of-work to proof-of-stake exacerbates these dynamics because the barriers to becoming a block producer are much lower.
+
+If searchers' bundles don't have privacy before being include then they risk having their bundles frontrun or unbundled and searchers will not want to send bundles to small block producers because of these risks. In turn this will be centralizing for the network because small block producers won’t be able to produce as profitable blocks as their large peers.
+
+In short: Flashbots Alpha democratized access to MEV revenue for miners, but it did so with limits. Bundle merging is complicated and there is a cap on the number of bundles that can be included and the way those bundles are ordered (by gas price, not overall profit). To produce the most profitable block possible, and compete with potentially superior proprietary solutions, those limits must be lifted. Furthermore, privacy is key to ensuring that every block producer has access to the most profitable block because it ensures that searchers can not be frontrun.
+
+## Beyond Flashbots Alpha: solutions to find the most profitable block
+To lift the limits of Flashbots Alpha we must find a solution that deals with the complexity of building the most profitable block and introduce privacy into the MEV marketplace. What solutions exist to do so?
```

Comment:
A subtle point, but on the first reason, I believe Flashbots Alpha already separates (part of) the building job from the miner--at least the most complicated part of bundle merging. I don't think PBS improves the status quo much in terms of the complexity of building, which already rests on the relay. In other words I think PBS solves the centralization issue (because miners still *could* build), but I think this it's only a limitation of Flashbots Alpha in that FB Alpha doesn't solve centralization, but not in terms of complexity of building.